### PR TITLE
parameter unit changes

### PR DESCRIPTION
**Pull request by simonmarchant**
Simple addition of a function. Please could at least one person see if it's ok.

### CODE REVIEW

File: PKPD/gui/mainWindow.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from PKPD.inference import inference as inf
 from PKPD.gui import abstractGui, home, simulation
 from PKPD.model import model as m
+import myokit
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py]: please move import Myokit to the top, i.e.

import myokit

from PKPD.....

File: PKPD/gui/mainWindow.py [hunk 2]
```diff
@@ -174,6 +175,16 @@ def next_tab(self):
                 # generate error message
                 error_message = 'The .csv file does not seem to be properly formatted. Please check again!'
                 QtWidgets.QMessageBox.question(self, 'Data structure not compatible!', error_message, QtWidgets.QMessageBox.Yes)
+
+            #Check Units in MMT file
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py]: to be consistent through out the program: space between # and comment, i.e.

`# check units in mmt file`

File: PKPD/gui/mainWindow.py [hunk 3]
```diff
@@ -174,6 +175,16 @@ def next_tab(self):
...
+            try:
+                self.model.model.check_units(mode=myokit.UNIT_STRICT)
+            except Exception as e: #Display Warning if Inconsistent
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py]: same here

File: PKPD/gui/mainWindow.py [hunk 4]
```diff
@@ -174,6 +175,16 @@ def next_tab(self):
...
+                warning_message = 'Warning: Units may be inconsistent'
+                QtWidgets.QMessageBox.question(self, warning_message, str(e),
+                                               QtWidgets.QMessageBox.Yes)
+
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py]: remove empty lines here, i.e. line 186 and 187

File: PKPD/gui/mainWindow.py [hunk 5]
```diff
@@ -190,6 +201,7 @@ def next_tab(self):
             QtWidgets.QMessageBox.question(self, 'Files not found!', error_message, QtWidgets.QMessageBox.Yes)
 
 
+
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py]: remove empty line 204. Functions/ methods are separated by 2 empty lines.

**Review by DavAug — CHANGES_REQUESTED** [review]
There is some stuff that has been introduced in the model repository branch. As I currently work on that branch, just ignore it and don't merge into the master just yet.

**Review by DavAug — APPROVED** [review]

**Comment by simonmarchant** [comment]
All approved and tests passing, merging now.

**Pull request merged**